.evergreen/build-mac.sh

```diff
@@ -9,7 +17,27 @@ for VERSION in 2.7 3.4 3.5 3.6 3.7 3.8; do
         PYTHON=/Library/Frameworks/Python.framework/Versions/$VERSION/bin/python3
     fi
     rm -rf build
-    $PYTHON setup.py bdist_wheel
+
+    # Install wheel if not already there.
```

Comment:
Should we add a FIXME/TODO in case this is expected to be fixed by an upcoming BUILD ticket?

Reply:
Good point, I opened https://jira.mongodb.org/browse/PYTHON-2375 and https://jira.mongodb.org/browse/BUILD-11842. Since it's tracked in Jira I don't think we need to add a TODO in code.